src/components/Nav/Nav.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import '../Nav/Nav.scss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faInstagram } from '@fortawesome/free-brands-svg-icons';
+import {
+  faMagnifyingGlass,
+  faCircleUser,
+  faBookmark,
+  faGear,
+} from '@fortawesome/free-solid-svg-icons';
+import { Link } from 'react-router-dom';
+function Nav() {
+  const [menuState, setMenuState] = useState(false);
+  const [searchState, setSearchState] = useState(false);
+
+  return (
+    <header className="Nav">
+      <div className="navLogoContainer">
+        <FontAwesomeIcon icon={faInstagram} className="navIcon" />
+        <p className="navLogo">westagram</p>
+      </div>
+      <div className="searchInputContainer">
+        <input
+          type="text"
+          placeholder="검색"
+          className="searchInput"
+          onFocus={() => {
+            setSearchState(true);
+          }}
+          onBlur={() => {
+            setSearchState(false);
+          }}
+        />
+        <FontAwesomeIcon icon={faMagnifyingGlass} />
+        {searchState && <SearchModal />}
+      </div>
+      <div className="navMenuContainer">
+        <img
+          src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/explore.png"
+          alt="explore"
+        />
+        <img
+          src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/heart.png"
+          alt="heart"
+        />
+        <img
+          src="https://s3.ap-northeast-2.amazonaws.com/cdn.wecode.co.kr/bearu/profile.png"
+          alt="profile"
+          className="navProfile"
+          onClick={() => {
+            setMenuState(!menuState);
+          }}
+        />
+        {menuState === true ? <UserMenu /> : null}
```

Comment:
논리곱 연산자를 활용한 단축평가로 다음과 같이 수정할 수 있겠네요!
```suggestion
        {menuState && <UserMenu />}
```